device/bgpv4.yaml

```diff
@@ -8,59 +8,134 @@ components:
   schemas:
     Device.Bgpv4:
       description: >-
-        Emulated bgpv4 protocol
+        Emulated BGPv4 router and routes
       type: object
       required: [name]
       properties:
         name: '../common/common.yaml#/components/x-inline/UniqueName'
         as_number_2_byte:
+          description: >-
+            Autonomous system (AS) number of 2 byte
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         dut_as_number_2_byte:
+          description: >-
+            Autonomous system (AS) number of the BGP peer router (DUT)
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_number_4_byte:
+          description: >-
```

Comment:
what happens when a user specifies as_number_2_bytes and as_number_4_bytes both?  

what happens to the DUT using 4 byte as number?

Reply:
At the time of discussion with protocol team it looks like modern DUTs are mainly support as_number_4_bytes. I have decide to expose only as_number and dut_as_number.